### DISCUSSION TITLE
“Approve this device” notification should have quick actions so it can be answered quickly without opening the app

### DISCUSSION THREAD

**Discussion by warpling** [community/community#65011] [Mobile]
<details>
<summary> currently, you have to tap the notification, open the app, tap approve on a dialogue. Ideally you could just pull down on the notification, press it, whatever, and tap approve and go back to what you were doing in the app that you wanted to sign into!</summary>
GitHub for iOS, Version 1.127.2 (108489730)
Device: iPhone15,2 (iOS 17.0)
</details>

**Comment by alcere** [comment]
Hi @warpling, thank you very much for your feedback! I will create an internal issue to evaluate and explore your proposal with our product team. Thanks again!
